src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -3395,42 +3287,6 @@ End Class
             return VerifyVB.VerifyCodeFixAsync(originalCode, fixedCode);
         }
 
-        [Fact]
-        public Task VB_Diagnostic_OverloadToken_WithConfigureAwait()
```

Comment:
Duplicates of `VB_Diagnostic_OverloadToken`.

Reply:
Same suggestion as above. I won't look at the deleted tests below in this same file, but you get the idea of matching the test with the name when possible instead of deleting it.